pyfixest/estimation/feols_compressed_.py

```diff
@@ -6,7 +6,15 @@
 import narwhals as nw
 import numpy as np
 import pandas as pd
-from tqdm import tqdm
+
+try:
+    from tqdm import tqdm
+except ImportError:
+
+    def tqdm(iterable, *args, **kwargs):
+        "Define a dummy tqdm function."
+        return iterable
```

Comment:
This is being repeated a lot ... why don't we let tqdm be a core dep? It seems very lightweight

Reply:
Same with `joblib`, actually - it has [no other dependencies except for python ](https://pypi.org/project/joblib/). More or less the motivation of this PR is to make the package so lightweight that it can easily be installed in closed environments (with no access to pypi / conda). My prior would be that both joblib and tqdm should actually be installed on every system that runs Python, though I'm not 100% sure. 

I am more or less gravitating towards having a default / core env that includes joblib, tqdm, GT etc that users install with `pip install pyfixest` and then a minimal but still functional env for those who can really only work with numpy, scipy, etc. Would be your suggestion to keep tqdm in the "minimal" dependency set @juanitorduz?